client/src/components/ui/personDetail.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+import Link from "next/link";
+
+interface personInfo {
+  personName: string;
+  personImg: string;
+  link?: string;
+}
+
+const PersonDetail = (props: personInfo) => {
+  return (
+    <Link href={props.link || ""}>
+      <div className="flex items-center gap-2 text-[17px] text-penni-text-regular-light-mode">
```

Comment:
`text-[17px]` try not defining custom sizes and stick to the preset tailwind classes so the spacing is not too inconsistent. I'm assuming Figma is just a reference and we don't need to follow it strictly. You can choose the next closest tailwind class.